### DISCUSSION TITLE
Is there a default config for standard ESP32 boards?

### DISCUSSION THREAD

**Discussion by shan-mcarthur** [moononournation/Arduino_GFX#280] [General]
Many ESP32 dev boards come with a standard SPI config of MOSI=23, MISO=19, and SCK=18, but I did not see any default config listed for that.

https://randomnerdtutorials.com/esp32-spi-communication-arduino/#esp32-spi-pins

**Comment by moononournation** [comment]
No unique default, you may check some pins_arduino.h in each variants folder:
https://github.com/espressif/arduino-esp32/tree/master/variants
All have its own definitions.
